src/pages/sc-controller/forms/StockCountControllerForm.js

```diff
@@ -0,0 +1,114 @@
+import { Grid } from '@mui/material'
+import { useContext, useEffect } from 'react'
+import * as yup from 'yup'
+import FormShell from 'src/components/Shared/FormShell'
+import toast from 'react-hot-toast'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useInvalidate } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { useForm } from 'src/hooks/form'
+import { ControlContext } from 'src/providers/ControlContext'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { SCRepository } from 'src/repositories/SCRepository'
+
+export default function StockCountControllerForm({ labels, maxAccess, recordId }) {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+
+  const invalidate = useInvalidate({
+    endpointId: SCRepository.StockCountController.page
+  })
+
+  const { formik } = useForm({
+    initialValues: {
+      recordId: null,
+      plantId: null,
+      name: ''
+    },
+    maxAccess,
+    enableReinitialize: true,
+    validateOnChange: true,
+    validationSchema: yup.object({
+      name: yup.string().required(),
+      plantId: yup.string().required()
```

Comment:
is plantId required because the plant combobox doesn't turn red and it doesn't have the "*"

![image](https://github.com/user-attachments/assets/9d35ee13-e99f-4b07-868b-9dff83eb0268)